src/server/services/validation.service.js

```diff
@@ -16,69 +16,84 @@ const {
 } = require("@slice-and-dice/register-a-food-business-validation");
 
 const errorMessages = {
-  declaration1: "You must tick all the declarations before continuing",
-  declaration2: "You must tick all the declarations before continuing",
-  declaration3: "You must tick all the declarations before continuing",
-  registration_role: "You must select a role before continuing",
-  operator_type: "You must select an operator type before continuing",
-  operator_first_name: "Not a valid first name",
-  operator_last_name: "Not a valid last name",
-  operator_town: "Not a valid town name",
-  operator_address_line_1: "Not a valid first line of address",
-  operator_address_line_2: "Not a valid second line of address",
-  operator_address_line_3: "Not a valid third line of address",
-  operator_postcode: "Not a valid postcode",
-  operator_postcode_find: "Not a valid postcode",
-  establishment_trading_name: "Not a valid establishment trading name",
-  operator_primary_number: "Not a valid operator phone number",
-  operator_secondary_number: "Not a valid operator phone number",
-  operator_email: "Not a valid operator email address",
-  contact_representative_name: "Not a valid representative name",
-  contact_representative_role: "Not a valid representative role",
-  contact_representative_number: "Not a valid representative phone number",
-  contact_representative_email: "Not a valid representative email address",
-  operator_company_name: "Not a valid company name",
+  declaration1: "Error You must tick all the declarations before continuing",
```

Comment:
Additionally, the suggestion in the report is for "Error: " to be 'visually hidden so it only appears for screen readers:  `<span class="govuk-visually-hidden">Error:</span> `